src/app/client/src/app/modules/core/components/main-header/main-header.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="ui grid" *ngIf="!showExploreHeader">
-    <div class="center aligned twelve column row">
+    <div class="center aligned twelve column row" [class.header-height-before-login]="!userService.userid">
         <div class="one wide tablet computer only column"></div>
-        <div class="two wide tablet computer only column padding-top-15">
+        <div class="two wide tablet computer only column" [class.padding-top-5]="!userService.userid" [class.padding-top-15]="userService.userid">
```

Comment:
need to remove `padding-top-5` class from here.

Reply:
Both suggestions are based on following - 

header has 80px height.. while logo has max 40px height.. padding top 5 will not align it in middle.